mobile/library/common/apple/utility.cc

```diff
@@ -0,0 +1,38 @@
+#include "library/common/apple/utility.h"
+
+namespace Envoy {
+namespace Apple {
+
+std::string toString(CFStringRef string) {
```

Comment:
I wonder if @RyanTheOptimist is familiar with these types from his time at Apple because I'm going to be fairly useless catching bugs in this fle.

Reply:
this code is mostly following the example in https://stackoverflow.com/a/9166500